### PR TITLE
Remove some warnings

### PR DESCRIPTION
**Pull request by matsl**
## What

Remove some warnings.

## Why

When looking at producing the set of warnings we have now I spotted a few easy fixes.  Each is in its own commit for easy review.

### CODE REVIEW

File: kotl/kexport.el [hunk 1]
```diff
@@ -462,7 +462,7 @@ used.  Also converts Urls and Klinks into Html hyperlinks.
 		 ">" "&gt;"
 		 (replace-regexp-in-string
 		      "<" "&lt;" (kview:label-separator kview))))
-	       i is-parent is-collapsible is-last-sibling no-sibling-stack level label contents)
```
Comment (matsl) [inline kotl/kexport.el:465]: Seems I added this while you removed using it so now I remove it.

File: hyrolo.el [hunk 2]
```diff
@@ -1424,7 +1424,6 @@ Return number of matching entries found."
 		  (re-search-backward hyrolo-entry-regexp nil t)
 		  (setq start (point))
 		  (re-search-forward hyrolo-entry-regexp nil t)
-		  (setq curr-entry-level-len (length (buffer-substring-no-properties start (point))))
```
Comment (matsl) [inline hyrolo.el:1427]: This is only set here but not used!?

Can this be code you are working on and that these var can be used later?

Comment (rswgnu) [reply hyrolo.el:1427]: You are correct.  This is code I inserted for compatibility with the in-development hynote.el and it is not yet finished.  Please ignore it for now and remove this change.

**Review by rswgnu — CHANGES_REQUESTED** [review]
All changes are fine except the one I commented on.  Remove that and we can apply this.

**Pull request merged**